### PR TITLE
Adding AMA banner to disability and appeals pages

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
Added the ability to display an AMA banner for certain pages when `appealsbanner` is set to true on content pages in `vagov-content` repo by adding `src/site/includes/appeals-banner.html` and created `if appealsbanner` statement in `src/site/layouts/page-breadcrumbs.html`

This was an internal request from the Appeals team in support of the launch for new decision review pages.

In order to view verify functionality, you will need the [AMA content feature branch](https://github.com/department-of-veterans-affairs/vagov-content/pull/229)

banner should display on following pages:
/disability/file-an-appeal/
/decision-reviews/

## Testing done
viewed on local build

## Screenshots

screen shot 2019-02-11 at 1 35 19 pm
screen shot 2019-02-11 at 1 35 19 pm

## Acceptance criteria
- [X] Banner displays when `appealsbanner` is set to true on content pages

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by 1Copenut** [comment]
@aubreyarcangel I did some initial testing on this PR, and have a question about intended behavior. Are you wanting screen readers to announce the **Appeals Process Has Changed** banner as an alert on immediate page load? Right now, of the following pairings, only one announces it as such:

* Win10 + IE11 + JAWS (does not announce)
* Win10 + Chrome + JAWS (does announce)
* Win10 + Firefox + NVDA (does announce)

I'll hold my PR review and comments until I've heard back RE this item.

File: src/site/includes/appeals-banner.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-warning">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">The Appeals process has changed</h3>
+        <div class="usa-alert-text">
+          <b>For VA decisions received aftger February 19, 2019</b>, please keep reading below to learn about your decision review options.
+          <br>
+          <br>
+          <b>For VA decisions received before February 19, 2019</b>, you'll need to follow a different process to appeal the decision.
+          <br>
+          <a href="#">Learn how to file an appeal</a>.
```
Comment (1Copenut) [inline src/site/includes/appeals-banner.html]: Is this being controlled with JavaScript, or should the `href` have a path?

File: src/site/includes/appeals-banner.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+  <div class="usa-alert-full-width usa-alert-full-width-warning">
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-warning">
```
Comment (1Copenut) [inline src/site/includes/appeals-banner.html]: Let's change the `aria-live` to "assertive". I've seen several industry leaders mark their alert boxes up that way. I also think we could move the `aria-live` and `role` attributes to the outer container, unless it causes a CSS issue.

File: src/site/includes/appeals-banner.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">The Appeals process has changed</h3>
+        <div class="usa-alert-text">
+          <b>For VA decisions received aftger February 19, 2019</b>, please keep reading below to learn about your decision review options.
```
Comment (1Copenut) [inline src/site/includes/appeals-banner.html]: It looks like "after" has an extra "g".

What do you think about wrapping the individual text blocks in `<p>` tags, and removing the `<br>` tags? It changes the spacing between blocks a touch, so please take that as a question to consider only.

**Review by 1Copenut — CHANGES_REQUESTED** [review]
@aubreyarcangel I requested a few small changes to the markup structure. Will pull code and review again when you're ready.

File: src/site/includes/appeals-banner.html (hunk 1 above)
Comment (aubreyarcangel) [reply src/site/includes/appeals-banner.html]: # was just a placeholder. I updated the `<a href>` tag to go link to `/decision-reviews/`

File: src/site/includes/appeals-banner.html (hunk 2 above)
Comment (aubreyarcangel) [reply src/site/includes/appeals-banner.html]: made requested changes.

File: src/site/includes/appeals-banner.html (hunk 3 above)
Comment (aubreyarcangel) [reply src/site/includes/appeals-banner.html]: Thanks for spotting that stay "g".

For the spacing, it's mostly how I've been handling spacing for the .md files. 

I attempted your advice and wrapped the text in a `<p>`, but it wasn't giving me the line breaks and spacing as intended, so put the `<br>`'s back in.

**Comment by 1Copenut** [comment]
@aubreyarcangel I took a quick look at the updates. When you've got a second, would you mind moving the `aria-live="assertive"` to the same div as the `role="alert"` ? They're complementary attributes, so having them at the same level shouldn't create any issues.

**Comment by 1Copenut** [comment]
@aubreyarcangel I wasn't completely settled with the previous markup. Screen readers were not honoring the alert banner as a region "block" like the banner, navigation, and main. I did some more digging and found that if we modify lines 1, 2, and 4, we can make the alert banner be a region stop:

```html
<!-- Change this outer container's role to region, add an aria-labelledby referencing H3 <ID> -->
<div
  class="usa-alert-full-width usa-alert-full-width-warning"
  role="region"
  aria-labelledby="appeal-heading"
>
  <!-- Move the aria-live="assertive" and role="alert" back to this first inner div -->
  <div class="usa-alert usa-alert-warning" aria-live="assertive" role="alert">
    <div class="usa-alert-body">
      <!-- Add an ID to the h3 -->
      <h3 class="usa-alert-heading" id="appeal-heading">The Appeals process has changed</h3>
      ...
</div>
```

I apologize for the third ask; this will be the last one and I'll approve straight away. Already tested and verified in these combinations:

* Win10 + IE11 + JAWS
* Win10 + Chrome + JAWS
* Win10 + Firefox + NVDA
* MacOS Mojave + Safari + VoiceOver
* iOS iPhone 8 + Safari + VoiceOver

**Review by 1Copenut — APPROVED** [review]

**Pull request merged**